yarn-project/sequencer-client/src/sequencer/public_processor.ts

```diff
@@ -193,7 +196,7 @@ export class PublicProcessor {
       throw new Error('Final public kernel was not executed.');
     }
 
-    const processedTx = makeProcessedTx(tx, finalKernelOutput, proof, revertReason);
+    const processedTx = makeProcessedTx(tx, finalKernelOutput, proof, publicRequests, revertReason);
```

Comment:
Hm, I think this `publicRequests` array ends up being ignored https://github.com/AztecProtocol/aztec-packages/pull/5599/files#diff-4a1e361c657ced7aec06972eec5f5fe475940baf9eee96e94997287348a60ca5R129

Reply:
Have fixed